cypress/plugins/common.js

```diff
@@ -23,41 +23,40 @@ const CONSTANTS = require('../support/constants/constants');
 
```

Comment:
Are the changes to this file required?

Reply:
if supportConfig.js doesn't exist it throwing an error and stop the execution.Particularly this change is required if there is no config module(default config module)